### PR TITLE
Disable all refreshes except graph refresh

### PR DESCRIPTION
**Pull request by borod108**
Only use graph refresh from now on.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1731037

### CODE REVIEW

**Comment by borod108** [comment]
or @agrare  :)

File: app/models/manageiq/providers/redhat/infra_manager/api_integration.rb [hunk 1]
```diff
@@ -24,8 +24,7 @@ def authentication_status_ok?(type = nil)
   end
 
   def use_graph_refresh?
-    Settings.ems_refresh.rhevm.try(:[], :inventory_object_refresh) &&
-      use_ovirt_sdk? && supported_api_versions.include?('4')
+    true
```
Comment (agrare) [inline app/models/manageiq/providers/redhat/infra_manager/api_integration.rb]: @borod108 can we remove this method and update all callers to just do what they would have done for the graph refresh case?

Comment (borod108) [reply app/models/manageiq/providers/redhat/infra_manager/api_integration.rb]: I could, I was thinking to do that together with removing tons of other code - you can see I only removed the specs to keep it safe...
Can we leave this to the big refactoring PR?

Comment (agrare) [reply app/models/manageiq/providers/redhat/infra_manager/api_integration.rb]: Okay

**Pull request merged**